From 1c26c5aa670df1419e76e6f9380bf01f7fec7f8e Mon Sep 17 00:00:00 2001
From: Rajesh Nyamagoud <nyamagoud@google.com>
Date: Fri, 13 May 2022 19:27:02 +0000
Subject: [PATCH] rust_defaults entry created for Keystore2 AIDL library.

A rust_defaults "keystore2_use_latest_aidl_rust" that includes the latest
Keystore2 AIDL library. Modules that depend on Keystore2 directly can include
this rust_defaults to avoid managing dependency versions explicitly

Test: Compile
Change-Id: Iabe9af743c4ceb787db42ff7cc619bab97ec55c4
---
 keystore2/aidl/Android.bp | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/keystore2/aidl/Android.bp b/keystore2/aidl/Android.bp
index ae08567d..d70f210c 100644
--- a/keystore2/aidl/Android.bp
+++ b/keystore2/aidl/Android.bp
@@ -199,3 +199,13 @@ cc_defaults {
         "android.system.keystore2-V2-ndk",
     ],
 }
+
+// A rust_defaults that includes the latest Keystore2 AIDL library.
+// Modules that depend on Keystore2 directly can include this rust_defaults to avoid
+// managing dependency versions explicitly.
+rust_defaults {
+    name: "keystore2_use_latest_aidl_rust",
+    rustlibs: [
+        "android.system.keystore2-V2-rust",
+    ],
+}
-- 
2.38.1

